triton-server/src/notification.ts

```diff
@@ -75,15 +75,15 @@ export const sendDatasetValidationStatusUpdateEmail = async () => {
                     },
                 )
             }
-            let body =
-                "A run has been validated: <br/>" +
+            const body =
+                "<b>A run has been validated:</b> <br/>" +
                 formattedData.map(
                     (dataset: ExtractedValidatedNotificationData) => {
                         return `<br/><br/>
-                            -   Run Name: ${dataset.projectAndRunInfo.run_name} <br/>
-                            -   Validated by: ${dataset.projectAndRunInfo.validated_by} <br/>
-                            -   Project: ${dataset.projectAndRunInfo.project_name}  ${dataset.projectAndRunInfo.project_id ?? ""} <br/>
-                            - Dataset/lane ${dataset.projectAndRunInfo.lane_number} status ${getValidationFlagLabel(dataset.projectAndRunInfo.validation_status)} <br/>
+                            -   <b>Run Name:</b> ${dataset.projectAndRunInfo.run_name} <br/>
+                            -   <b>Validated by:</b> ${dataset.projectAndRunInfo.validated_by} <br/>
+                            -   <b>Project:</b> ${dataset.projectAndRunInfo.project_name}  ${dataset.projectAndRunInfo.project_id ?? ""} <br/>
```

Comment:
We should reduce the space between `-` and `<b>` to one space. so like `-  <b>Run Name...`